bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestColumnDescriptorAdapter.java

```diff
@@ -191,7 +191,10 @@ public void badExpressionLessThan(){
 
   @Test
   public void testBlankExpression(){
-    HColumnDescriptor result = adapter.adapt("family", asColumnFamily(""));
-    Assert.assertEquals(new HColumnDescriptor("family"), result);
+    // An empty gc expression means keep all versions, which is not supported by HBase.
+    expectedException.expectMessage("Maximum versions must be positive");
+    expectedException.expect(IllegalArgumentException.class);
```

Comment:
Should this be an error case, or should we allow 0 = infinite when creating a column family?